dev/release/post-03-website.sh

```diff
@@ -115,7 +140,6 @@ This is a major release covering more than ${rough_n_development_months} months
 This release includes ${n_commits} commits from ${n_contributors} distinct contributors.
 
 \`\`\`console
-$ ${contributors_command_line}
```

Comment:
Do you know why `${contributors_command_line}` was called here but then `${contributors}` was used below? I tried to work out a reasoning why but couldn't quite grok what was going on here.

Reply:
Originally there were two variables:

1. the code to run to get the list of commuters and number of commits
2. the output of running the code from 1

I think it was done like that before so that the variable 1 could be displayed in the output.  However, now we use longer commands to generate the list of committers/commits, it's no longer desirable to display it and so this has been simplified by just using a single variable.

Reply:
Ok, cool makes sense thanks!